src/test/resources/regressions/issues/000228.gobra

```diff
@@ -0,0 +1,71 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package types
+
+type TSlice []int
```

Comment:
maybe you could duplicate this test case but use type aliases instead of type declarations

Reply:
Done! In the process of doing this, I found a bug which makes me doubt that it was ever possible to instantiate a value of an alias type ^^ this should be fixed now